docs/infra/api-markdown-documenter/admonition-node.mjs

```diff
@@ -4,7 +4,7 @@
  */
 
 //@ts-check
-/** @typedef {import("@fluid-tools/api-markdown-documenter").DocumentationNode} DocumentationNode */
+/** @typedef {import("@fluid-tools/api-markdown-documenter").BlockContent} BlockContent */
```

Comment:
Note: the Documentation Domain has been updated to be more restrictive in terms of what kinds of contents can go under different kinds of parent nodes. [Block Content](https://github.com/syntax-tree/mdast?tab=readme-ov-file#content-model) is the appropriate kind of child for our Admonition nodes.